backend/routes/auth.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+const router = express.Router();
+
+const Food = require("../model/foodSchema");
+
+router.get("/", (req, res) => {
+  res.send("Home");
+});
+
+router.get("/get-food",async (req, res) => {
+  try {
+    const data = await Food.find({});
```

Comment:
`Food.find({})` can be replaced with `Food.find()` and also it is a good practice to remove all `console.log` before merging the code to production.